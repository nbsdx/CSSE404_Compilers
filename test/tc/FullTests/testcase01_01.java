class A1 {
    public static void main(String[] a2) {
	System.out.println(1);
    }
}

class A3 {
    public int a4(int a5) {
	return a5+1;
    }
}

class A6 extends A3 {
    public boolean a7() {
	return this.a5(4)==5;
    }
}
