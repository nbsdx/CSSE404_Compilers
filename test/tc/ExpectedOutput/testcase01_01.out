No method named a5 found for class A6
The operand types, null and int, are not compatible for equality comparison
